File: Java Basics - January 2020/WorldSwimmingRecord.java
import java.util.Scanner;

public class WorldSwimmingRecord {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        double recordInSeconds = Double.parseDouble(scanner.nextLine());
        double distanceInMetres = Double.parseDouble(scanner.nextLine());
        double oneMetrePerSecond = Double.parseDouble(scanner.nextLine());

        double neededDistanceToSwim = distanceInMetres * oneMetrePerSecond;

        double minus15m = distanceInMetres/15;
        double minus15m2 = minus15m * 12.5;

        double totalTime = neededDistanceToSwim +minus15m2;

        if (totalTime > recordInSeconds){
            double diff = totalTime - recordInSeconds;
            System.out.printf("Yes, he succeeded! The new world record is %.2f seconds.", totalTime);
        }else {
            double diff2 = recordInSeconds - totalTime;
            System.out.printf("No, he failed! He was %.2f seconds slower.");
        }

    }
}

File: Python Basics - October 2020 /Conditionals/world_swimming_record.py
import math
record_in_seconds = float(input())
distance_in_metres = float(input())
time_in_sec_for_meter = float(input())

sum_time_in_seconds = distance_in_metres * time_in_sec_for_meter
slowing_distance_metres = (math.floor(distance_in_metres / 15)) * 12.5

total_time = slowing_distance_metres + sum_time_in_seconds

waste = abs(record_in_seconds - total_time)

if total_time < record_in_seconds:
    print(f"Yes, he succeeded! The new world record is {total_time:.2f} seconds.")
elif total_time >= record_in_seconds:
    print(f"No, he failed! He was {waste:.2f} seconds slower.")

